lib/matplotlib/__init__.py

```diff
@@ -283,6 +283,12 @@ def wrapper():
 _ExecInfo = namedtuple("_ExecInfo", "executable version")
 
 
+class ExecutableUnavailableError(FileNotFoundError):
```

Comment:
Should this maybe be called `ExecutableNotFoundError` in analogy to `FileNotFoundError`?

Reply:
Yeah, that makes sense.

Reply:
I guess I went with `Unavailable` b/c the motivating case was when an executable `convert` is found, but `convert --version` returns nonsense.